### PR TITLE
Feat picture uploader

### PR DESCRIPTION
**Pull request by Dldvlpr**
Feature: Now we can upload a picture.
When you edit the picture of a art the old fades away to make way for the new.
When you delete an art the picture are delete too.

### CODE REVIEW

File: templates/artCard/index.html.twig [hunk 1]
```diff
@@ -9,7 +9,7 @@
             {% for artCard in artCards %}
                 <div class="card artist-card col-6 col-md-4 d-flex flex-column  p-1 p-lg-2">
                     <img class="card-img img-fluid img-thumbnail bg-black"
-                         src="{{ artCard.pictureArt }}" alt="This is the profil picture of an artist"/>
+                         src="/uploads/artPicture/{{ artCard.pictureArt }}" alt="This is a picture of an art"/>
```
Comment (aldaitzwild) [inline templates/artCard/index.html.twig:12]: No you can't have the directory hard coded in the twig

Comment (Dldvlpr) [reply templates/artCard/index.html.twig]: Thats ok i use a twig global for call folder

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Try to implement the upload with the same trick we use in the herakles workshop

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Clean the global twig

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**